store/k8s/k8s_store.go

```diff
@@ -0,0 +1,36 @@
+package k8s
+
+import (
+	"context"
+	"github.com/libopenstorage/secrets"
+	"github.com/libopenstorage/secrets/k8s"
+	"github.com/libopenstorage/secrets/store"
+)
+
+type k8sStore struct {
+	secrets secrets.Secrets
+}
+
+func (s *k8sStore) Get(ctx context.Context, key store.SecretKey) (secret map[string]any, err error) {
+	keyContext := map[string]string{secrets.KeyVaultNamespace: key.Prefix}
```

Comment:
This should not be KeyVaultNamespace since that constant is for vault secrets provider.